_topic_maps/_topic_map_rosa.yml

```diff
@@ -1960,14 +1960,6 @@ Topics:
   Topics:
   - Name: Backup and restore by using VM snapshots
     File: virt-backup-restore-snapshots
-  - Name: Installing and configuring OADP
-    File: virt-installing-configuring-oadp
-  - Name: Backing up and restoring virtual machines
-    File: virt-backup-restore-overview
-  - Name: Backing up virtual machines
-    File: virt-backing-up-vms
-  - Name: Restoring virtual machines
-    File: virt-restoring-vms
```

Comment:
Same comment as above. To avoid future merge conflicts and problems cherrypicking forward, please update this ROSA topic map to include the commented out lines from `main` https://github.com/openshift/openshift-docs/pull/75004/files#diff-234616541d1acc030cb26887a6e3cdee267c4b17adfe71116e1460fc0c628e23R1899-R1900

Reply:
Thanks. Changes made